js/main.js

```diff
@@ -35,6 +42,18 @@ Site = {
   onResize: function() {
     var _this = this;
 
+    _this.resizeDelay();
+  },
+
+  resizeDelay: function() { 
```

Comment:
possibly should use a debounce: https://www.paulirish.com/2009/throttled-smartresize-jquery-event-handler/